src/mumble/MainWindow.cpp

```diff
@@ -2738,7 +2738,7 @@ void MainWindow::on_gsSendTextMessage_triggered(bool down, QVariant scdata) {
 }
 
 void MainWindow::on_gsSendClipboardTextMessage_triggered(bool down, QVariant) {
-	if (!down) {
+    if (!down || (QApplication::clipboard()->text().isEmpty())) {
```

Comment:
The line should be indented with a tab (as it was before).

Should I fix it for you?

Reply:
Give me a minute to try to fix that, it would give me an opportunit to learn how amending commits interacts with pull requests

Reply:
No problem.

Reply:
It's fixed on my end--a tab in line with all the other tabs, but the preview on GitHub still looks shady to me. Feel free to fix it if it's still bad, at least then I'll know what to look for.

Reply:
should be fixed now